Package.swift

```diff
@@ -209,8 +209,7 @@ let package = Package(
           .when(platforms: [.windows])),
         .linkedLibrary(
           "jvm",
-          .when(platforms: [.linux, .macOS, .windows])
-        ),
+          .when(platforms: [.iOS, .macOS, .tvOS, .watchOS, .macCatalyst, .linux, .openbsd, .wasi, .windows])),
```

Comment:
We’re not really promising compatibility with a bunch of those yet

Reply:
I'd suggest removing: wasi, wasm, catalyst, ios, tvos, watchos -- it's not platforms we support right now

```suggestion
          .when(platforms: [.macOS, .linux,])),
```

wdyt?